src/index.js

```diff
@@ -1,11 +1,23 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+// Styles
 import './index.css';
-import App from './App';
+
+// Components
+import App from './components/App';
+
+// Reducers
+import reducer, { DEFAULT_STATE } from './actions/reducers/index';
+
+const store = createStore(reducer, DEFAULT_STATE);
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Provider store={store}>
+      <App />
+    </Provider>
   </React.StrictMode>,
   document.getElementById('root'),
 );
```

Comment:
- You needed to add initial/sample books here [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/InitialBooks.png)]